audio/art/art.license

```diff
@@ -0,0 +1,11 @@
+License
+
+The music was downloaded from Youtube
+
+# License Agreement Explained.
+What You Can Do anything with Our Sounds:
+You can do anything with our sound file.
+We don't have any responsibility about this sound file.
+ 
+Email : supunedirisuriya2002@gmail.com
+Github : STEdirisuriya
```

Comment:
I think, the license which reflects your intention most would be public domain.
Just for the record, in Germany, you can not give away all your rights and public domain is not possible. As such one would CC0.